content/en/blog/2022/k8sattributes.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Improved troubleshooting using k8s attributes
+linkTitle: Kubernetes attributes
+date: 2022-06-27
+author: Ruben Vargas
+spelling:
+  cSpell:ignore k8sattributes k8sattributesprocessor K8sattributes k8sprocessor
+  cSpell:ignore K8sprocessor KUBE
+---
+
+Attaching Kubernetes resource metadata to OpenTelemetry traces can be very
+useful because it lets you identify which resource (such as a pod) is failing or
+having performance problems. It is also generally useful for correlating across
+other signals.
+
+In this article, you'll learn how to configure the OpenTelemetry collector to
+use the `k8sattributesprocessor` processor in different scenarios. At the end,
+we present some alternatives.
+
+Details of the OpenTelemetry collector pipeline won't be covered in this post.
+For those details please, refer to
+[OpenTelemetry collector documentation](/docs/collector/).
+
+## How k8s attributes are attached
+
+At a high level, k8s attributes are attached to traces as
+[resources](/docs/concepts/glossary/#resource). This is for two reasons:
+
+1.  K8s attributes fit the definition of what a resource is -- an entity for
+    which telemetry is recorded
+2.  It centralizes this metadata, which is relevant for any generated span.
+
+Let's dive in and see how to do it!
+
+## Using k8sattributes processor
+
+The `k8sattributesprocessor` is an OpenTelemetry collector processor that
```

Comment:
this sentence is redundant, since the next sentence says the same thing, you might remove it or merge them

> The `k8sattributesprocessor` is an OpenTelemetry collector processor that automatically discovers pod attributes and attaches those ...